src/features/app/dashboard/api/index.ts

```diff
@@ -9,7 +9,7 @@ export const useGetDashboard = (params: DocRequestParams) => {
   return useSuspenseQueries({
     queries: [
       {
-        queryKey: ["users", getUserParams],
+        queryKey: [`user_${getUserParams.documentId}`, getUserParams],
```

Comment:
getUserParamsにdocumentId含まれてるので
```suggestion
        queryKey: ["users", getUserParams],
```

Reply:
あ、そういうことか！すません！
修正しました〜。